### PR TITLE
docs(aio): tech edits to upgrade-lazy

### PR DESCRIPTION
**Pull request by kapunahelewong**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ x] No
```

## Other information

These are tech edits to the copy.

### CODE REVIEW

File: aio/content/guide/upgrade-lazy.md [hunk 1]
```diff
@@ -8,43 +8,45 @@
...
 
-  The [Change Detection](guide/upgrade#change-detection) sub-section only applies to applications
-  that use `UpgradeModule`. Change detection is handled differently with `downgradeModule()`.<br />
-  We still recommend reading the sub-section in order to better understand the differences and their
-  implications.
+  The [Change Detection](guide/upgrade#change-detection) section of 
+  [Upgrading from AngularJS](guide/upgrade) only applies to apps 
+  that use `UpgradeModule`. You handle change detection differently 
+  with `downgradeModule()`. The [Change Detection](guide/upgrade#change-detection)
+  section clarifies the differences and their implications while
```
Comment (gkalpak) [inline aio/content/guide/upgrade-lazy.md]: The `Change Detection` section of upgrade does not clarify (or even mention) the diffrences.

A little background:
There are two ways to upgrade (which have many similarities but a few important differences - one of which is how change detection works). One way is using `UpgradeModule` (described in `upgrade.md`). The second is using `downgradeModule()` (described in this guide, with references to `upgrade.md` for the things work the same in both).

This is what I am trying to say here:

The `Change Detection` section in `upgrade.md` describes how change detection works in hybrid apps that use `UpgradeModule` (it doesn't discuss or mention any differences). The differences for app that use `downgradeModule()` are discussed in this guide (`upgrade-lazy.md`).
Having read the `Change Detection` section in `upgrade.md` (i.e. knowing how change detection works in apps with `UpgradeModule`), will help the reader understand the differences and their implications (discussed below).

File: aio/content/guide/upgrade-lazy.md [hunk 2]
```diff
@@ -55,85 +57,84 @@ As mentioned before, one of the key differences between `downgradeModule()` and
 to do with change detection and how it is propagated between the two frameworks.
 
 With `UpgradeModule`, the two change detection systems are tied together more tightly. Whenever
-something happens in the AngularJS part of the application, change detection is automatically
-triggered on the Angular part and vice versa. This is convenient as it ensures that no important
-change is missed by either framework. Most of the time, though, these extra change detection runs
-are unnecesary.
+something happens in the AngularJS part of the app, change detection is automatically
+triggered on the Angular part and vice versa. This is convenient as it ensures that neither framework misses an important change. Most of the time, though, these extra change detection runs are unnecesary.
```
Comment (gkalpak) [inline aio/content/guide/upgrade-lazy.md]: _unnecesary_ --> _unnecessary_ :grin:

File: aio/content/guide/upgrade-lazy.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Upgrading from AngularJS (in a more flexible way)
+# Migrating from AngularJS to Angular
```
Comment (gkalpak) [inline aio/content/guide/upgrade-lazy.md]: The tough part is that we have two upgrade guides, which talk about the same thing (upgrading/migrating from AngularJS to Angular). The difference is in whether you use `UpgradeModule` or `downgradeModule()`. Both do the same thing (allow creating a hybrid app) and the differences are quite low-level (so not a good fit for a title).

Maybe we could incorporate this guide into the other one?

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/content/guide/upgrade-lazy.md (hunk 1 above)
Comment (kapunahelewong) [reply aio/content/guide/upgrade-lazy.md]: Thank you for the explanation - I wasn't getting it before. How about:

"The Change Detection section of Upgrading from AngularJS only applies to apps that use UpgradeModule. Though you handle change detection differently with downgradeModule(), which is the focus of this guide, reading the Change Detection section provides helpful context for what follows."

File: aio/content/guide/upgrade-lazy.md (hunk 2 above)
Comment (kapunahelewong) [reply aio/content/guide/upgrade-lazy.md]: Fixed.

File: aio/content/guide/upgrade-lazy.md (hunk 3 above)
Comment (kapunahelewong) [reply aio/content/guide/upgrade-lazy.md]: As per an offline conversation, @gkalpak and I decided to keep it separate for the time being.

File: aio/content/guide/upgrade-lazy.md (hunk 1 above)
Comment (gkalpak) [reply aio/content/guide/upgrade-lazy.md]: Sounds good 👍

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
I somehow messed this up. Cherry-picked the commit (as [ff7159fd4][1]) into [my PR][2].

[1]: https://github.com/gkalpak/angular/commit/ff7159fd40b4801cc470f530f287b38aa9f0e137
[2]: https://github.com/angular/angular/pull/18487/commits/ff7159fd40b4801cc470f530f287b38aa9f0e137

**Pull request closed without merging**